src/h2/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM openjdk:8-alpine
```

Comment:
Wondering, is this a JDK-based or JRE-based image? The JDK-based ones are much bigger and not really needed, so just calling it out.